python/lsst/ctrl/mpexec/execFixupDataId.py

```diff
@@ -60,52 +60,24 @@ def assoc_fixup():
     dimensions : `str` or sequence [`str`]
         One or more dimension names, quanta execution will be ordered
         according to values of these dimensions.
-    reverse : `bool`, optional
-        If `False` (default) then quanta with higher values of dimensions
-        will be executed after quanta with lower values, otherwise the order
-        is reversed.
     """
 
     def __init__(self, taskLabel: str, dimensions: Union[str, Sequence[str]], reverse: bool = False):
         self.taskLabel = taskLabel
         self.dimensions = dimensions
-        self.reverse = reverse
```

Comment:
You did not remove it from parameter list? Why `reverse` is not supported, is it impossible in a new world?

Reply:
Added back in